src/Polly.Testing/Polly.Testing.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net7.0;net6.0;netstandard2.0;net472;net462</TargetFrameworks>
```

Comment:
I wonder if we need quite so many TFMs for this package given it's brand new and doesn't necessarily need the same backwards compatibility "baggage".

Reply:
I think just having `netstandard2.0` can be completely fine here.